sycl/include/CL/sycl/detail/cg.hpp

```diff
@@ -150,22 +150,22 @@ class CG {
 
   /// Type of the command group.
   enum CGTYPE : unsigned int {
-    NONE = 0,
-    KERNEL = 1,
-    COPY_ACC_TO_PTR = 2,
-    COPY_PTR_TO_ACC = 3,
-    COPY_ACC_TO_ACC = 4,
-    BARRIER = 5,
-    BARRIER_WAITLIST = 6,
-    FILL = 7,
-    UPDATE_HOST = 8,
-    RUN_ON_HOST_INTEL = 9,
-    COPY_USM = 10,
-    FILL_USM = 11,
-    PREFETCH_USM = 12,
-    CODEPLAY_INTEROP_TASK = 13,
-    CODEPLAY_HOST_TASK = 14,
-    ADVISE_USM = 15,
+    __SYCL_NONE = 0,
```

Comment:
```suggestion
    None = 0,
```
?

Reply:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-caps

Reply:
Hi, are you suggesting to change uppercase to lowercase (for example, __SYCL_NONE --> None)?

Reply:
Yes. It's more readable and solves the collision problem.

Reply:
It's called CamelCase.

Reply:
ADVISE_USM --> AdviseUSM

Reply:
To be more specific, we are following LLVM's coding standards. 
Here is relevant section - https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly

Reply:
fixed